acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/signing/BlsSigningAcceptanceTest.java

```diff
@@ -86,6 +89,29 @@ public void ableToSignUsingHashicorp() {
     }
   }
 
+  @Test
+  @EnabledIfEnvironmentVariables({
+    @EnabledIfEnvironmentVariable(named = "AZURE_CLIENT_ID", matches = ".*"),
+    @EnabledIfEnvironmentVariable(named = "AZURE_CLIENT_SECRET", matches = ".*"),
+    @EnabledIfEnvironmentVariable(named = "AZURE_KEY_VAULT_NAME", matches = ".*"),
+    @EnabledIfEnvironmentVariable(named = "AZURE_TENANT_ID", matches = ".*")
+  })
+  public void ableToSignUsingAzure() {
+    final String clientId = System.getenv("AZURE_CLIENT_ID");
+    final String clientSecret = System.getenv("AZURE_CLIENT_SECRET");
+    final String tenantId = System.getenv("AZURE_TENANT_ID");
+    final String keyVaultName = System.getenv("AZURE_KEY_VAULT_NAME");
+    final String secretName = "TEST-KEY";
+
+    final String configFilename = keyPair.getPublicKey().toString().substring(2);
+    final Path keyConfigFile = testDirectory.resolve(configFilename + ".yaml");
+    metadataFileHelpers.createAzureYamlFileAt(
+        keyConfigFile,
+        new AzureSigningMetadata(clientId, clientSecret, tenantId, keyVaultName, secretName));
```

Comment:
Seems unnecessary to create a AzureSigningMetadata, could just pass the params through since we just pull all values out into a map again to create the yaml file.

Reply:
applied.